### ISSUE TITLE
learning_rate not being read in/config not truly being parsed

### ISSUE THREAD

**Issue by jshermeyer** [eriklindernoren/PyTorch-YOLOv3#298]
As far as I can tell the learning_rate and decay are not actually being read in from the config file to tweak the adam optimizer. 

Despite changing the learning rate to 0.0001 and specifying decay in the config when I print(optimizer) in train.py: 

Adam (
Parameter Group 0
    amsgrad: False
    betas: (0.9, 0.999)
    eps: 1e-08
    **_lr: 0.001_**
    **_weight_decay: 0_**
)

Any comment on this?

**Comment by jshermeyer** [comment]
For the record this can be fixed easily if you modify line 77 to: 

`optimizer = torch.optim.Adam(model.parameters(), lr=float(model.hyperparams['learning_rate']), weight_decay=float(model.hyperparams['decay']))`

I understand that the Adam optimizer does modify the learning rate on it's own, however it's disingenuous to have a learning_rate or decay in the config file and then not use it.
